Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Pass/Specific/SwapChainPass.h

```diff
@@ -36,7 +35,7 @@ namespace AZ
             AZ_RTTI(SwapChainPass, "{551AD61F-8603-4998-A7D1-226F03022295}", ParentPass);
             AZ_CLASS_ALLOCATOR(SwapChainPass, SystemAllocator, 0);
 
-            SwapChainPass(const PassDescriptor& descriptor, const WindowContext* windowContext);
+            SwapChainPass(const PassDescriptor& descriptor, const WindowContext* windowContext, const WindowContext::SwapChainMode swapchainMode);
```

Comment:
nit: To keep consistent the naming of swap chain as 2 words: `swapchainMode` -> `swapChainMode`